pkg/kubectl/resource_printer.go

```diff
@@ -322,13 +322,20 @@ type handlerEntry struct {
 type PrintOptions struct {
 	NoHeaders          bool
 	WithNamespace      bool
+	WithKind           bool
 	Wide               bool
 	ShowAll            bool
 	ShowLabels         bool
 	AbsoluteTimestamps bool
+	KindName           string
 	ColumnLabels       []string
 }
 
+type PrintKindOptions struct {
```

Comment:
This seems redundant with the fields added to PrintOptions, which are publically accessible. Why the extra copy?